Code/GraphMol/ChemReactions/Wrap/testReactionWrapper.py

```diff
@@ -521,5 +521,85 @@ def test21CheckRawIters(self):
       mb2 = Chem.MolToMolBlock(reactants[i])
       self.assertEquals(mb1, mb2)
 
+  def test22RunSingleReactant(self):
+    # from
+    # A Collection of Robust Organic Synthesis Reactions for In Silico Molecule Design
+    # Markus Hartenfeller,*, Martin Eberle, Peter Meier, Cristina Nieto-Oberhuber,
+    # Karl-Heinz Altmann, Gisbert Schneider, Edgar Jacoby, and Steffen Renner
+    # Novartis Institutes for BioMedical Research, Novartis Pharma AG, Forum 1,
+    # Novartis Campus, CH-4056 Basel, Switzerland Swiss Federal Institute of Technology (ETH)
+    #  Zurich, Switzerland
+    smirks_thiourea = "[N;$(N-[#6]):3]=[C;$(C=S):1].[N;$(N[#6]);!$(N=*);!$([N-]);!$(N#*);!$([ND3]);!$([ND4]);!$(N[O,N]);!$(N[C,S]=[S,O,N]):2]>>[N:3]-[C:1]-[N+0:2]"
+    rxn = rdChemReactions.ReactionFromSmarts(smirks_thiourea)
+    reagents = map(Chem.MolFromSmiles, ['C=CCN=C=S', 'NCc1ncc(Cl)cc1Br'])
```

Comment:
Better to use a list comprehension here: `reagents = [Chem.MolFromSmiles(x) for x in ('C=CCN=C=S', 'NCc1ncc(Cl)cc1Br')]`.
If you're going to stick with `map`, you need to explicitly convert it to either a list or a tuple in order for this code to work with Python3